### ISSUE TITLE
Does the youtube player handle vimeo videos?

### ISSUE THREAD

**Issue by davidkneely** [PlayWithMagic/PlayWithMagic.org#119]

**Comment by pkarjala** [comment]
No; Vimeo videos need to be handled differently.

The full shebang on Vimeo embeds can be found at https://vimeo.com/help/faq/sharing-videos/embedding-videos

**Comment by marknelsonengineer** [comment]
This issue should include basic video URL validation.

**Comment by pkarjala** [comment]
Agreed; a simple regex should take care of that, unless there's a Java
library that will do it for us.

---

Patrick Karjala
pkarjala@gmail.com
